enterprise/backend/src/metabase_enterprise/serialization/v2/load.clj

```diff
@@ -46,6 +51,14 @@
                     (throw e)))))]
       (reduce loader ctx deps))))
 
+(defn- path-error-data [error-type expanding path]
+  (let [last-model (:model (last path))]
+    {:path       path
+     :deps-chain expanding
+     :model      (some-> last-model name)
+     :table      (some-> last-model t2/table-name)
```

Comment:
Unfortunately it's a model name, so it needs to be converted to a keyword first...

Reply:
Does that mean there's no reason to be calling `name` for `:model` then?

Reply:
yeah, it's a string. This is the reason why all serdes multimethods dispatch on strings rather than keywords btw. Even though it feels silly.